classes/music_album.rb

```diff
@@ -0,0 +1,16 @@
+require_relative './item'
+
+class MusicAlbum < Item
+  def initialize(genre, publish_date, on_spotify: false)
+    @on_spotify = on_spotify
+    @genre = genre
+    @publish_date = publish_date
+    super(publish_date, genre: @genre)
+  end
```

Comment:
Please send the params to the super (Item class) directly and do not create a new variable.